src/components/NodeLink/functionUpdateVis.js

```diff
@@ -298,11 +300,22 @@ function updateVis() {
   link = linkEnter.merge(link);
 
   link.classed("muted", false);
-
   link
     .select("path")
-    .style("stroke-width", 3)
-    .style("stroke", edgeColor)
+    .style("stroke-width", d => {
+      if (linkWidthVariable) {
+        return d[linkWidthVariable]
+      } else{
+        return 3
+      }
+    })
+    .style("stroke", d => {
+      if (linkColorVariable !== null) {
+        return linkColorScale(d[linkColorVariable])
+      } else{
+        return "#888888"
+      }
+    })
```

Comment:
This is great, though we will also want to support continuous colormaps as well. That can certainly be added in a future PR.

Reply:
Opened #57